bin/node/src/service.rs

```diff
@@ -142,19 +140,20 @@ pub fn new_partial(
         client.clone(),
     );
 
-    let metrics_or_error = match config.prometheus_registry() {
-        Some(register) => Metrics::new(register).map_err(|err| format!("{:?}", err)),
-        None => Err("registry not available".to_string()),
+    let metrics = match config.prometheus_registry() {
+        Some(register) => match Metrics::new(register) {
+            Ok(metrics) => metrics,
+            Err(e) => {
+                warn!("Failed to register Prometheus metrics:{:?}", e);
+                Metrics::noop()
+            }
+        },
+        None => {
+            info!("Running without the metrics not available");
```

Comment:
Nit: this sentence doesn't quite parse. :P

Reply:
Thanks, updated.